### PR TITLE
added Request form api

### PR DESCRIPTION
**Pull request by ChiragJS**
need to remove RequestDTO class, no use of it . update the put mapping in the next Pr 
made an urgent pr !

### CODE REVIEW

File: src/main/java/org/ieeervce/gatekeeper/PDFNotConversionException.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.ieeervce.gatekeeper;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class PDFNotConversionException extends Exception{
```
Comment (chrisvrose) [inline src/main/java/org/ieeervce/gatekeeper/PDFNotConversionException.java:7]: FailedFileConversionException?

If you want to keep the same name, pls add a javadoc mentioning what this exception means

File: src/main/java/org/ieeervce/gatekeeper/controller/RequestFormController.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            }
+        }
+        catch(ItemNotFoundException e ){
+            throw new InvalidDataException("Invalid user Id");
+            }
+        catch (java.io.IOException e){
+            throw new PDFNotConversionException("Could not store pdf");
+        }
+
+        return requestFormService.save(requestForm);
```
Comment (chrisvrose) [inline src/main/java/org/ieeervce/gatekeeper/controller/RequestFormController.java:71]: This returns the entire thing as JSON. how is the file returned?
If as a json array, that will make the response too huge, could you check?

**Review by chrisvrose — COMMENTED** [review]
minor things to take a look at.

Overall seems in good shape.

**Comment by SundarakrishnanN** [comment]
ignore this pr,made a newer one with slightly cleaner code

**Comment by chrisvrose** [comment]
Please close if not required.

**Pull request closed without merging**